frontend/src/routes/pipelines/project.ts

```diff
@@ -0,0 +1,13 @@
+import { routePipelineDetails } from '~/routes/pipelines/global';
+
+const globProjects = 'projects';
```

Comment:
might want this one in a `src/routes/projects` folder -- as it's not Pipeline specific.